tools/azure-pipelines/jobs-template.yml

```diff
@@ -100,6 +100,8 @@ jobs:
         module: tests
       misc:
         module: misc
+      connector_hive_312:
+        module: connector_hive_312
```

Comment:
I think any connector should be part of the connector tests, not as a separate module.

Reply:
Agreed.